src/configs/consts.ts

```diff
@@ -0,0 +1,9 @@
+export const breakpoints = { xs: 520, sm: 768, md: 1024, lg: 1280, xl: 1640 };
+
+export const PRODUCT_IMAGES_BASE_PATH = '/images/products';
+
+export const categories = [
+  { name: 'Куртки', value: 'jackets' },
+  { name: 'Футболки', value: 'tshirts' },
+  { name: 'Аксесуары', value: 'accesories' },
+];
```

Comment:
названия констант следует писать капсом
```suggestion
export const BREAKPOINTS = { xs: 520, sm: 768, md: 1024, lg: 1280, xl: 1640 };

export const PRODUCT_IMAGES_BASE_PATH = '/images/products';

export const CATEGORIES = [
  { name: 'Куртки', value: 'jackets' },
  { name: 'Футболки', value: 'tshirts' },
  { name: 'Аксесуары', value: 'accesories' },
];
```

Reply:
Понял.